docs/CHANGELOG.asciidoc

```diff
@@ -63,6 +64,36 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+
+[[release-notes-8.13.3]]
+== {kib} 8.13.3
+
+The 8.13.3 release includes the following bug fixes.
+
+[float]
+[[fixes-v8.13.3]]
+=== Bug Fixes
+
+Alerting::
+* Manage loading fields at initialization ({kibana-pull}180412[#180412]).
+Elastic Security::
+For the Elastic Security 8.13.3 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Fixes managed agent policy preconfiguration update ({kibana-pull}181624[#181624]).
+* Use lowercase dataset in template names ({kibana-pull}180887[#180887]).
+* Fixes KQL/kuery for getting Fleet Server agent count ({kibana-pull}180650[#180650]).
+Lens & Visualizations::
+* Fixes table sorting on time picker interval change in *Lens* ({kibana-pull}182173[#182173]).
```

Comment:
@lcawl this fixes table sorting https://github.com/elastic/kibana/pull/182173 didn't make it into BC2 of 8.13.3 which is getting released tomorrow. Thanks!  cc @dej611

Reply:
Thanks for letting me know! Fixing in https://github.com/elastic/kibana/pull/182370